### PR TITLE
fix: allow identifier to contain dot

### PR DESCRIPTION
**Pull request by pwmcintyre**
https://github.com/fraugster/parquet-go/issues/15

### CODE REVIEW

File: parquetschema/schema_def.go [hunk 1]
```diff
@@ -85,8 +85,8 @@ func SchemaDefinitionFromColumnDefinition(c *ColumnDefinition) *SchemaDefinition
 //	digit ::= '0' | '1' | '2' | '3' | '4' | '5' | '6' | '7' | '8' | '9'
 //	time-unit ::= 'MILLIS' | 'MICROS' | 'NANOS'
 //	boolean ::= 'false' | 'true'
-//	identifier ::= <alpha> <alphanum>*
-//	alpha ::= 'a'..'z' | 'A'..'Z'
+//	identifier ::= <alpha><alphanum>*
```
Comment (pwmcintyre) [inline parquetschema/schema_def.go:88]:
```suggestion
//	identifier ::= <alpha> <alphanum>*
```

**Comment by pwmcintyre** [comment]
marked as draft: I found that while it parses the schema without error, it's fails to write the file

eg
```shell
$ docker run -it --rm -v ${PWD}:/data  nathanhowell/parquet-tools cat /data/schema_issue/output.parquet
org.apache.parquet.io.InvalidRecordException: meta not found in message spark_schema {
  optional binary version (STRING);
  optional binary meta.id (STRING);
}
```

**Comment by fzerorubigd** [comment]
dot has a special meaning in the parquet field name. it shows the nested structure since we store each column in a row group and in headers we have the path to that column, parquet reader splits the name with dots to get the path.

**Comment by pwmcintyre** [comment]
@fzerorubigd — thanks for the clarification. I really like your package, but it appears incompatible with our quirky schema :(

I will close the PR / Issue

**Pull request closed without merging**